_oasis

```diff
@@ -82,6 +83,27 @@ Library "irmin-unix"
                  git.unix, cohttp.lwt
   Modules:       Irmin_unix
 
+Library "irmin-krypto"
+  Path:             lib/krypto
+  Findlibname:      krypto
+  FindlibParent:    irmin
+  BuildDepends:     irmin, nocrypto
+  Modules:          Irmin_krypto, Irmin_krypto_cipher, Irmin_krypto_km
+
+Library "irmin-link"
+  Path:             lib/link
+  Findlibname:      link
+  FindlibParent:    irmin
+  BuildDepends:     irmin, irmin.mem, irmin.unix, irmin.fs
+  Modules:          Irmin_link
```

Comment:
One of the reason to split `Irmin_link` into the various other backends would be to avoid depending on `irmin.unix` here. Might be other ways to avoid this (for instance by having a functor of `Irmin.FS.IO`). In any case, we don't want to depend on `*.unix` to make it compile to a unikernel later.